bitrise.yml

```diff
@@ -33,10 +40,28 @@ workflows:
     - TEST_CERTIFICATE_URL: $TEST_CERTIFICATE_URL
     - TEST_CERTIFICATE_URL_PASSWORD: $TEST_CERTIFICATE_URL_PASSWORD
     steps:
-    - go-list:
-    - golint:
-    - errcheck:
-    - go-test:
+    - script:
```

Comment:
these steps did not work for me with newer go version

Reply:
I think these validations should remain, and maybe need to fix the lint/errcheck errors

Reply:
the validation remained but these steps did not work I've created a script to run these validation